README.md

```diff
@@ -169,7 +169,12 @@ Please refer to [Mapbox Maps SDK for iOS](platform/ios/platform/ios/) for detail
 
 ```bash
 cd platform/ios
+# open macOS project in Xcode
+make xproj
```

Comment:
While wrapping commands in a makefile is nice for tests it makes the whole build system even more convoluted in my opinion. We should revisit this in the future.

Reply:
@ntadej  — did you mean for this comment in `platform/ios/Makefile`.  L172,L173 is for macOS developers (which I had to fix some stuff in.

> While wrapping commands in a makefile is nice for tests it makes the whole build system even more convoluted in my opinion. We should revisit this in the future.